### PR TITLE
Fix: slider-rating-modal code

### PR DESCRIPTION
**Pull request by Divya70**
Fix slider , modal and rating code

### CODE REVIEW

**Review by Tejinder15 — COMMENTED** [review]
you can remove type="button" attribute from the button tag

image

File: pages/Model/model.html [hunk 1]
```diff
@@ -85,13 +85,18 @@ <h1 class="modal-heading">Spiderman</h1>
...
+                        style="width: 100%; 
+                        height: 300px;
+                         border:0; 
+                         margin: 2rem;
+                         transform: scale(1);
+                          overflow:hidden;" sandbox="allow-scripts allow-same-origin">
```
Comment (NamrataRaikwar2002) [inline pages/Model/model.html:99]: you can remove these inline styling.

File: pages/documentation.css [hunk 2]
```diff
@@ -92,6 +92,10 @@
     --new-badge-bg: #ff3f6ccc;
     --new-badge-color: #fff;
 
+    /* Rating Color */
+    --rating-color: #ff7b00;
+    --rating-hover-color: #ffa500;
```
Comment (NamrataRaikwar2002) [inline pages/documentation.css:97]: you can use variables for colours.

**Review by Husain01 — APPROVED** [review]

**Review by NamrataRaikwar2002 — APPROVED** [review]

File: pages/documentation.css (hunk 2 above)
Comment (Divya70) [reply pages/documentation.css:97]: namarata these are var which i define

File: pages/Model/model.html (hunk 1 above)
Comment (Divya70) [reply pages/Model/model.html:99]: this comes by default

**Pull request merged**